swmm-toolkit/CMakeLists.txt

```diff
@@ -14,9 +14,10 @@ cmake_minimum_required (VERSION 3.17)
 project(swmm-toolkit)
 
 
-set(Python_FIND_VIRTUALENV FIRST)
-set(Python_ROOT_DIR "/usr/local/anaconda3")
-find_package(Python 3.7 COMPONENTS Development Interpreter REQUIRED)
+set(Python_EXECUTABLE ${PYTHON_EXECUTABLE})
+set(Python_INCLUDE_DIRS ${PYTHON_INCLUDE_DIR})
+set(Python_LIBRARY ${PYTHON_LIBRARY})
+find_package (Python ${PYTHON_VERSION_STRING} COMPONENTS Interpreter Development)
```

Comment:
Where are the python environment variables set? Are they set as part of the install process?

Reply:
Yeah, the python variables get set when scikit-build calls cmake.